src/dhtproto/node/request/Listen.d

```diff
@@ -91,7 +92,14 @@ public abstract scope class Listen : SingleChannel
             }
             finally
             {
-                this.reader.fiber.epoll.unregister(disconnect_detector);
+                static if (hasFeaturesFrom!("ocean", 3, 6))
```

Comment:
Did you try using [ocean's `hasMethod`])https://github.com/sociomantic-tsunami/ocean/blob/f7e5805c5c327058eb0c913820727c898bbd16d5/src/ocean/meta/traits/Aggregates.d#L79)? That way, it wouldn't be a breaking change.